pkg/authconfigmap/authconfigmap_test.go

```diff
@@ -343,22 +343,22 @@ var _ = Describe("AuthConfigMap{}", func() {
 		}
 
 		It("should add a role and a user", func() {
-			cm := addAndSave(roleA, RoleNodeGroupUsername, RoleNodeGroupGroups)
-			cm = addAndSave(userA, userAUsername, userAGroups)
+			_ = addAndSave(roleA, RoleNodeGroupUsername, RoleNodeGroupGroups)
```

Comment:
How about re-arrange check statements like below

```suggestion
			cm := addAndSave(roleA, RoleNodeGroupUsername, RoleNodeGroupGroups)
			Expect(cm.Data["mapRoles"]).To(MatchYAML(expectedRoleA))
			cm = addAndSave(userA, userAUsername, userAGroups)
			Expect(cm.Data["mapUsers"]).To(MatchYAML(expectedUserA))
```